### PR TITLE
Use specific_bootmenu_params from bootloader_setup on s390x arch

### PR DESCRIPTION
**Pull request by mimi1vx**
(poo#18282)

Note - DUD and INSTALLER_SELF_UPDATE url must be shorter than 52
characters for s390x parmfile

### CODE REVIEW

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -187,22 +187,23 @@ sub bootmenu_network_source {
...
-        # 'ifcfg=*=dhcp' sets this variable in ifcfg-eth0 as well and we can't
-        # have them both as it's not deterministic.
-        $netsetup = get_var("NETWORK_INIT_PARAM", "ifcfg=*=dhcp SetHostname=0");
-        $args .= " $netsetup autoyast=" . data_url(get_var("AUTOYAST")) . " ";
-    }
-    else {
-        $netsetup = " " . get_var("NETWORK_INIT_PARAM") if defined get_var("NETWORK_INIT_PARAM");    #e.g netsetup=dhcp,all
-        $args .= $netsetup;
+    my $args = "";
+    if (check_var('ARCH', 's390x')) {
```
Comment (dzedro) [inline lib/bootloader_setup.pm]: wrong condition :)

**Comment by mimi1vx** [comment]
http://argus.suse.cz/tests/550

**Comment by mimi1vx** [comment]
@Soulofdestiny please review and try zkvm :)

**Comment by Soulofdestiny** [comment]
verification runs on zkvm:
yast_self_update: http://opeth.suse.de/tests/5118
yast_no_self_update: http://opeth.suse.de/tests/5117

so LGTM for me, but there is one space too much which I would like to see fixed

**Comment by okurz** [comment]
it's better than before… not perfect … but better :-)

**Pull request merged**